public/diagnostic.js

```diff
@@ -23,3 +23,29 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+  addToCart: [],//Ember.thingy(),
+  totalPrice: []//Math.sum(orderPrice);
+});
+
+const Order = Ember.Object.extend({
+  unitPrice: [],
+  quantity: [],
+  orderPrice: Ember.computed('unitPrice', 'quantity', function(){
+    return this.get('unitPrice') * this.get('quantity');
+  });
+});
+
+let cart = Cart.create({
+  unitPrice: 2,
+  quantity: 20,
```

Comment:
A `Cart` doesn't have either a `unitPrice` or a `quantity` -- those are properties that `Orders` have.